client/src/containers/ProfilePage/index.tsx

```diff
@@ -39,16 +39,13 @@ const ProfilePage = ({ id }: { id: string }) => {
 		activity,
 	});
 	const [user, setUser] = useState(userData);
-	const { editMode, isLoading } = user;
-
+	const { isLoading } = user;
+	const [editMode, setEditMode] = useState<string>('');
+	const [isLoadAdditioanl, setIsLoadAdditional] = useState<boolean>(true);
 	const isCurrentUser = currentUser ? id === currentUser.id : false;
 
 	const showManager = (modeToShow: string) => {
```

Comment:
```suggestion
	const showManager = (modeToShow: enum) => {
```
It is better to use enum except of simple string